library_management/permissions.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import permissions
+
+
+class IsStaffEditorPermission(permissions.DjangoModelPermissions):
+    def has_permission(self, request, view):
+        user = request.user
+        if user.is_librarian:
+            if user.has_perm("library_management.add_book"):
+                return True
+            if user.has_perm("library_management.view_book"):
+                return True
+            if user.has_perm("library_management.delete_book"):
+                return True
+            if user.has_perm("library_management.change_book"):
+                return True
+
+        if user.is_librarian == False:
```

Comment:
If the above condition fails this condition would always be true. We can replace this redundant check with an `else`.